data/strings/strings.txt

```diff
@@ -5149,49 +5149,125 @@
     zh-Hant = 在螢幕上顯示
 
   [pref_map_style_title]
-    comment = Settings «Map» category: «Night style» title
-    tags = android,ios
-    en = Night Mode
-    af = Nagmodus
-    ar = الوضع الليلي
-    az = Gecə rejimi
-    be = Начны рэжым
-    bg = Нощен режим
-    ca = Mode nocturn
-    cs = Noční režim
-    da = Nattilstand
-    de = Nachtmodus
-    el = Νυχτερινή λειτουργία
-    es = Modo nocturno
-    et = Öörežiim
-    eu = Gaueko modua
-    fa = حالت شب
-    fi = Yötila
-    fr = Mode nuit
-    he = מצב לילה
-    hi = रात का मोड
-    hu = Éjszakai üzemmód
-    id = Mode Malam
-    it = Modalità notturna
-    ja = 夜間モード
-    ko = 나이트 모드
-    lt = Nakties režimas
-    mr = रात्र मोड
-    nb = Nattmodus
-    nl = Nachtmodus
-    pl = Tryb nocny
-    pt = Modo noturno
-    pt-BR = Modo noturno
-    ro = Mod nocturn
-    ru = Ночной режим
-    sk = Nočný režim
-    sv = Nattläge
-    th = โหมดกลางคืน
-    tr = Gece Modu
-    uk = Нічний режим
-    vi = Chế độ ban đêm
-    zh-Hans = 夜间模式
-    zh-Hant = 夜間模式
+    comment = Settings «Map» category: «Appearance» title
+    tags = android,ios
+    en = Appearance
+    ar = المظهر
+    az = Görünüş
+    be = Выгляд
+    bg = Изглед
+    ca = Aspecte
+    cs = Vzhled
+    da = Udseende
+    de = Erscheinungsbild
+    el = Εμφάνιση
+    es = Aspecto
+    fi = Ulkoasu
+    fr = Apparence
+    hi = प्रकटन
+    hu = Megjelenés
+    id = Tampilan
+    it = Aspetto
+    ja = 外観
+    ko = 모양
+    lt = Išvaizda
+    mr = प्रतिमा
+    nb = Utseende
+    nl = Weergave
+    pl = Wygląd
+    pt = Apresentação
+    pt-BR = Aparência
+    ro = Aspect
+    ru = Оформление
+    sk = Vzhľad
+    sv = Utseende
+    th = รูปแบบ
+    tr = Görünüş
+    uk = Вигляд
+    vi = Giao diện
+    zh-Hans = 外观
+    zh-Hant = 外觀
+
+  [pref_appearance_light]
+    comment = Settings "Appearance" category: "Light" title
+    tags = android,ios
+    en = Light
+    ar = فاتح
+    az = Parlaq
+    be = Светлы
+    bg = Светъл
+    ca = Clar
+    cs = Světlý
+    da = Lys
+    de = Hell
+    el = Ανοιχτόχρωμη
+    es = Claro
+    fi = Vaalea
+    fr = Claire
+    he = בהיר
```

Comment:
This `he` is good. Curious, how come you have it for this string but not the other two?